app/components/ViewContainer.js

```diff
@@ -1,19 +1,50 @@
 import React, { Component, PropTypes, View } from 'react-native';
 import AR from './AR.js';
 import Map from './Map.js';
+import DropNewPinButton from '../containers/container_dropNewPin'
 
 export default class ViewContainer extends Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
     this.state = {
-      view: 'map'
+      view: 'map',
+      currLoc: null,
```

Comment:
Should these be in the state of this component, or in the redux store? It looked like we were maintaining a value of current location in the Map component as well; although there it was called 'lastPosition'. What's the difference between those two things?  Shouldn't this just be in one place and then passed down?